code/modules/research/rdconsole.dm

```diff
@@ -709,7 +709,7 @@ Nothing else in the console has ID requirements.
 		display_name = "<A href='?src=[REF(src)];view_node=[node.id];back_screen=[screen]'>[display_name]</A>"
 	l += "<div class='statusDisplay technode'><b>[display_name]</b> [RDSCREEN_NOBREAK]"
 	if(minimal)
-		l += "<br>[node.description]"
+		l += ""
```

Comment:
If you're not adding anything here now, you should get rid of the whole if. And remove the minimal parameter if it's not used anywhere else.